### DISCUSSION TITLE
Is this lovelace UI and its map usable with official native integration?

### DISCUSSION THREAD

**Discussion by TE-it** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#744] [General]
Hi,

I noticed that there is an official recommendation to switch away from the HACS integration of roborock https://github.com/humbertogontijo/homeassistant-roborock, see latest readme:
![grafik](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/assets/171480607/a07e402d-ec37-4d82-85d5-7a7054090533)

This means for me the great lovelace-xiaomi-vacuum-map-card will not work any more. At least I could not find a way to show the live map of the robot.  
Is there any recommended platform which works with the native roborock integration: https://www.home-assistant.io/integrations/roborock/
![grafik](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/assets/171480607/7feedf55-6026-4dc9-8bc2-1d8375b9ef11)

Thanks for your input.

**Comment by PiotrMachowski** [comment]
I think that at this moment you can keep using the custom integration until something breaks for you.

Process of adding support for official Roborock integration is on the way: https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/issues/691

**Reply by TE-it** [reply]
Oh sorry I did not see this upcoming feature. I highly appreciate that this is already on way.
Yep, the usual question: When will it be ready? Are we talking about few weeks or months?

I did already remove the other stuff because I just started on HA. I prefer the clean solution rigth from the beginning.

**Reply by PiotrMachowski** [reply]
And the usual answer: I have no idea 😄 it's really hard to tell anything, I don't work on it as regularly as before

**Reply by TE-it** [reply]
Answer is accepted :smiley:
Good luck regarding the challenges ahead and thanks for your work!
